talon/signature/bruteforce.py

```diff
@@ -12,31 +12,47 @@
 log = logging.getLogger(__name__)
 
 # regex to fetch signature based on common signature words
-RE_SIGNATURE = regex.compile(r'''
+ENG_GER_CHARS_SPACES = '[a-z\u00E4\u00F6\u00FC\u00C4\u00D6\u00DC\u00df\s]'
```

Comment:
can you also add the PR title on the top of the file?  e.g., `# [ch14047] improve talon signature extraction`